### ISSUE TITLE
"having field = value" in select statement can't work correctly

### ISSUE THREAD

**Issue by nivalsoul** [Anchormen/sql4es#33]
if I have the sql like:  select name, count(1) as num from table group by name having num = 5
the "num = 5" can not  work correctly because of this use 【=】comparison operation on two Double type values, the code is in the following file:
sql4es/src/main/java/nl/anchormen/sql4es/model/expression/SimpleComparison.java

> **if(this.comparisonType == Type.EQUAL) return leftValue == rightValue;**
> if(this.comparisonType == Type.GREATER_THAN) return leftValue > rightValue;
> if(this.comparisonType == Type.GREATER_THAN_OR_EQUAL) return leftValue >= rightValue;
> if(this.comparisonType == Type.LESS_THAN) return leftValue < rightValue;
> if(this.comparisonType == Type.LESS_THAN_OR_EQUAL) return leftValue <= rightValue;

if this can use: return leftValue.equals(rightValue); ?

**Comment by corneversloot** [comment]
You are right that this poses a problem and I think your fix is suffice. I will see if I can update it soon.

ps would be better to implement the HAVING as a pipeline aggregation and let ES handle it but that would be future work.

**Comment by corneversloot** [comment]
I have added the proposed fix which is released in version 0.9.2.4. Please test if this version works for you as it should
